firebase.json

```
@@ -143,6 +143,7 @@
       { "source": "/reference/widgets/widgetindex", "destination": "/reference/widgets", "type": 301 },
       { "source": "/release/breaking-changes/scrollable_alert_dialog", "destination": "/release/breaking-changes/scrollable-alert-dialog", "type": 301 },
       { "source": "/release/breaking-changes/win_lifecycle_process_function", "destination": "/release/breaking-changes/win-lifecycle-process-function", "type": 301 },
+      { "source": "/release/breaking-changes/material-3-migration", "destination": "/release/breaking-changes/material-3-migration", "type": 301 },
```

Thanks for adding a redirect! This one seems to redirect to itself though.

I'd consider something simple:

```suggestion
      { "source": "/material-3-migration", "destination": "/release/breaking-changes/material-3-migration", "type": 301 },
```

(And then move to be in the proper order)